crates/storage/codecs/Cargo.toml

```diff
@@ -20,6 +20,7 @@ value-256 = ["codecs-derive/value-256"]
 bytes.workspace = true
 codecs-derive = { path = "./derive", default-features = false }
 revm-primitives = { workspace = true, features = ["serde"] }
+alloy-primitives = { workspace = true, features = ["rand", "rlp"] }
```

Comment:
I don't like this, we should stick to reexported types for internal crates

Reply:
@mattsse Done